hippunfold/workflow/rules/compose_warps.smk

```diff
@@ -93,4 +92,11 @@ rule compute_warp_jacobian:
     group: 'subj'
     script: '../scripts/compute_jacobian.py'
 
+rule compute_graddev:
+    """ Generic rule for computing graddev from jacobian"""
+
+    input: '{prefix}_jacobian.nii.gz'
+    output: '{prefix}_grad_dev.nii.gz'
```

Comment:
minor point, but should rename suffix to `graddev` instead of `grad_dev` to be bids compliant (suffix should only have letters/numbers)